fhservices.cfg.j2

```diff
@@ -57,26 +57,26 @@ define host {
        host_name {{ rhmap_router_dns }}
 }
 
-{% for fhservice in fh_services_ping %}
+{% for fhservice in fh_services if 'ping' in fhservice.get("checks") %}
```

Comment:
> wdyt?

I think if I was maintaining this and came across a problem, I'd prefer to see a KeyError rather than maybe a TypeError.
In the case where we do `... in ...`, we'd get a TypeError, but in other places it would just produce the string "None" in the nagios config, not so easy to catch.